pkgs/by-name/pi/pixelorama/package.nix

```diff
@@ -35,6 +35,17 @@ stdenv.mkDerivation (finalAttrs: {
     godot
   ];
 
+  # Pixelorama is tightly coupled to the version of Godot that it is meant to be built with,
+  # and Godot does not follow semver, they break things in minor releases.
+  preConfigure = ''
+    godot_ver="${lib.versions.majorMinor godot.version}"
+    godot_expected=$(sed -n -E 's/.*Pixelorama uses Godot ([0-9]+\.[0-9]+).*/\1/p' README.md)
```

Comment:
Perhaps `config/features=PackedStringArray("4.4")` in [`project.godot`](https://github.com/Orama-Interactive/Pixelorama/blob/ad2393fc026a61238558925c759db3094f77ef1c/project.godot#L20) would be better to match against.

Reply:
I have no idea about Godot tbh, so, I saw this line, but wasn’t sure about it. If you think it’s ok to match on it, then sure.